rafs/src/metadata/direct_v6.rs

```diff
@@ -30,6 +32,7 @@ use std::slice;
 use std::sync::Arc;
 
 use arc_swap::{ArcSwap, Guard};
+use std::cell::RefCell;
```

Comment:
This line could be merged with line 20 above.

Reply:
fixed